web-client/src/views/FileHandlingHelpers/cloneFile.error.test.ts

```diff
@@ -29,6 +29,7 @@ describe('cloneFile', () => {
   it('should fail when attempting to clone something other than a file', async () => {
     let error;
     try {
+      // @ts-ignore
```

Comment:
I can probably just get rid of this test given the typing should enforce the intended behavior.